src/utils/Polyfills.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Some Polyfills required for cross browser compatibility.
+ */
+
+// IE does not support outerHTML property on SVGElement
+// http://stackoverflow.com/questions/12592417/outerhtml-of-an-svg-element
+Object.defineProperty(SVGElement.prototype, 'outerHTML', {
```

Comment:
If you choose this solution, it will always override the outerHTML function of the SvgElements